charts/gateway-operator/Chart.yaml

```diff
@@ -8,7 +8,7 @@ maintainers:
 name: gateway-operator
 sources:
   - https://github.com/Kong/charts/tree/main/charts/gateway-operator
-version: 0.2.3
+version: 0.3.0
```

Comment:
I'd say that we don't want to release this just yet. We can accrue changes up until 1.4.0 is out and we change the default version to 1.4.0 and only release 0.3.0 version of the chart then.

WDYT?

Reply:
Releasing the Helm charts is cheap, thus I decided to do it in a single step. Having it released makes testing smoother as you don't have to clone the charts repo locally.

Reply:
That makes sense. 👍